source/04_sdk/02_reference/01_client_module/07_contribution/index.rst

```diff
@@ -0,0 +1,154 @@
+Contribution Endpoint
+---------------------
+
+.. warning:: WIP
+
+
+The ``DefaultRestClient`` includes a ``DefaultRestContributionEndpoint`` 
+with the following functionalities.
+
+
+Save contribution
+^^^^^^^^^^^^^^^^^
+
+Saving a contribution is done using the ``saveContribution`` method. The idea is that the client already works with an instance of the contribution. Client currently supports only composition inside contribution with ``create/update/delete`` change types. The creation of ``ContributionCreateDto`` explained below.
```

Comment:
It seems hard to understand what "works with" means.
Wpuld it not be sufficient to say that it accepts a ContributionCreateDto, which can be created manually, or with the ContributionBuilder?
Not sure, is it the client that only supports Compositions, or is this a constraint of the `ContributionBuilder` and `ContributionCreateDto`s for other types could be created manually?

Reply:
changed for first question. ContributionBuilder supports the composition only. For creating manually CompositionCreateDto  should be provided data as  "openEHR Reference Model" required